app/views/welcome/show.html.erb

```diff
@@ -11,7 +11,7 @@
       <% end %>
 
       <% if flash[:alert] %>
-        <%= render "govuk_publishing_components/components/error_alert", { message: flash[:alert] } %>
+        <%= render "govuk_publishing_components/components/error_alert", { message: sanitize(flash[:alert]) } %>
```

Comment:
Interesting, how come we don't need `I18n.t(` here?

Reply:
The flash message is already a normal string